invenio_stats/tasks.py

```diff
@@ -11,9 +11,26 @@
 
 from celery import shared_task
 from dateutil.parser import parse as dateutil_parse
+from datetime import timedelta
 
 from .proxies import current_stats
 
+StatsEventTask = {
+    "task": "invenio_stats.tasks.process_events",
+    "schedule": timedelta(minutes=30),
+    "args": [("record-view", "file-download")],
+}
+StatsAggregationTask = {
+    "task": "invenio_stats.tasks.aggregate_events",
+    "schedule": timedelta(hours=1),
```

Comment:
I haven't changed the frequency. In theory, it could run more often, since it will only do the changes from the previous run (instead of evaluating the whole day as before)